### ISSUE TITLE
Counter-Strike Source: backend crash

### ISSUE THREAD

**Issue by jose1711** [obozrenie/obozrenie#7]
2017-02-09 22:43:28 | Core/Core | Internal backend error for Counter-Strike: Source.

tested with git version from february 9th 2017. arch linux.

**Comment by vorot93** [comment]
Thanks for the report.

Steam titles in Obozrenie are not entirely stable at the moment. They use QStat (which is almost abandonware) to query Steam master server. And the ground has shifted as latest attempts to manually communicate with / debug hl2master.steampowered.com (old) protocol have failed. I may be missing something but, alas, Valve promotes the cathedral of Steam and its API, Steamworks. [1]

TL;DR it may be time to either rewrite Steam title support or drop them from Obozrenie altogether.

[1]: See the note at https://developer.valvesoftware.com/wiki/Master_Server_Query_Protocol
